bin/webpack.js

```diff
@@ -66,6 +68,7 @@ const CLIs = [
 	{
 		name: "webpack-command",
 		package: "webpack-command",
+		binName: "webpack",
```

Comment:
Could you change this to `webpack-command`?

Reply:
Sure, keeping in mind that changing it to `webpack-command` will mean the user will need webpack-command>=0.3.0 installed for it to work, since that bin name was added in https://github.com/webpack-contrib/webpack-command/pull/40.